### ISSUE TITLE
Messages from injectable_generator don't make sense in 0.3.0

### ISSUE THREAD

**Issue by ResoDev** [Milad-Akarie/injectable#35]
Although it's not an error, I constantly get a message saying:
```
Injectable Generator ---------------------------------------------------

[AuthBloc] depends on [IAuthFacade] which is not injectable!
[SignInFormBloc] depends on [IAuthFacade] which is not injectable!
[NoteActorBloc] depends on [INoteRepository] which is not injectable!
[NoteFormBloc] depends on [INoteRepository] which is not injectable!
[NoteWatcherBloc] depends on [INoteRepository] which is not injectable!
[FirebaseAuthFacade] depends on [FirebaseAuth] which is not injectable!
[FirebaseAuthFacade] depends on [GoogleSignIn] which is not injectable!
[NoteRepository] depends on [Firestore] which is not injectable!

Did you forget to annotate the above classe(s) or their implementation with @injectable?
------------------------------------------------------------------------
```

Well, of course, that `IAuthFacade` is not injectable. It's an abstract class 😉 (and yes, its implementation is registered).

Classes like `FirebaseAuth` are marked `@lazySingleton` but this may stem from #34

**Comment by Milad-Akarie** [comment]
Can you show me an implementation of a class that's reported missing?

**Comment by sterrenburg** [comment]
On 0.3.0, I receive these kinds of messages for all my singletons:

```
[WARNING] injectable_generator:injectable_config_builder on lib/injection.dart:
FetchSummary(g<ApiRepository>())
[WARNING] injectable_generator:injectable_config_builder on lib/injection.dart:
PingPihole(g<ApiRepository>())
[WARNING] injectable_generator:injectable_config_builder on lib/injection.dart:
StorageService.localStorage(g<LocalStorage>())
...
```

Not sure if they relate to these `not injectable` warnings, but my messages only report 'WARNING',

**Comment by Milad-Akarie** [comment]
@sterrenburg ironically that's just a forgotten print statement lol

**Comment by Milad-Akarie** [comment]
@ResoDev  fixed in injectable_generator 0.3.2
Thanks for reporting
